Kolskid standomix300 make pasta draft

dmbrasw2 opened the pull request:
## Completed Work

- Finished procedure: p-standomix300-make-pasta
- Expanded on previously-drafted steps
- Added pictures
- Added notes

## User Scenario
"Anna’s grandmother used to make homemade pasta for the family. Now, Anna wants to make homemade pasta herself, but knows from her grandmother that properly rolling out pasta sheets and cutting the noodles is time consuming and difficult to freehand. She wants to ease into the journey of making pasta like her grandmother by using a pasta attachment for her stand mixer."

## Notable Review Requests:
 - The sheeting step gets repetitive. Would like an opinion on using "repeat steps x-y" [Right here](https://github.com/ENG517/Procedure/commit/2189241626e7ec2272d4f961685750d32fa40d13#r165624173)

 - Decided to include a recipe yield section. Unsure about the [placement and formatting.](https://github.com/ENG517/Procedure/commit/2189241626e7ec2272d4f961685750d32fa40d13#r165624230)

 - Unsure about "1 1/2" vs "1.5"-type formatting for measurements.
 
 - Overall I'm pretty happy with the whole thing, but I'm open to feedback about verbiage and clarity

sezito2 commented on watsonj/watson-installing-matplotlib.md:
```diff
@@ -0,0 +1,35 @@
+
+## Step 1: Install Matplotlib
+
+You can install Matplotlib by opeing your computer's terminal and running the following command:
```
Start with imperative phrases -- Install Matplotlib by ...

sezito2 replied:
Start with imperative phrases -- Install Matplotlib by ...

sezito2 commented on watsonj/watson-installing-matplotlib.md:
```diff
@@ -0,0 +1,35 @@
+
+![Python2](assets/images/screenshots/Python2.png)
+
+## Step 1: Install Matplotlib
```
Since the procedure is shorter, it might be more efficient to remove these heading and just make a numbered list with the steps instead

sezito2 reviewed (approved):
Effective procedure that is easy to read through. I think some edits to the coaching portion would make it more useful. I like your inclusion of pictures.

Ademola-Adepoju reviewed (commented):
I like it, Justin.

If you're going to stick with the subheadings, i think you might want to add something useful to close out the article. Right now, it looks like an abrupt end. That's my only suggestion.

dmbrasw2 committed:
Initial setup

Copied and renamed TOCOPY

dmbrasw2 committed:
Made branch 1

Renamed task 1 to make pasta and made a make pasta branch

dmbrasw2 committed:
Rough draft

Rough draft of the overall pasta procedure.  Will tweak and refine later.
Still need pics and links, and plenty of rewriting.

dmbrasw2 committed:
Finished Pasta

Reworked and expanded on the pasta procedure and added more steps.

Added pictures.

dmbrasw2 committed:
Pull request created

Created and filled out the pull request file.

Tweaked some spelling in the procedure file.

dmbrasw2 committed:
Edited writing and found a new picture

Found a picture of the stand mixer's speed lever. Not sure if I will add it, but I at least I have the option
Resized and cropped the pasta sheeting pic

Tweaked a lot of grammar and word choice

Revised the pasta sheeting section

dmbrasw2 committed:
Alt text for pics

Added alt text for the pictures

dmbrasw2 committed:
Minor wording edits

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
 ## What You Will Need
 ### Tools
```
I like how you put these sections in the beginning and made them very easy to view (designed nicely)

alyssajohnsonunofficial replied:
Agreed, the sections are succinct and well organized!

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
+1. Measure out the flour and salt, and set out the eggs. 
+2. Install the beater in the mixer
 
 ###  Make the dough
```
I think imperative action verbs would be nice for these action titles

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
-6. Wrap in plastic wrap and allow to rest for __ minutes
+1. Add the eggs, flour, salt, and 2 tablespoons of water in the stand mixer.
+2. Beat on speed 2 until combined (about 30 seconds.)
+3. Swithc out the beater for the dough hook.
```
small typo 'switch'

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
+4. Set to speed 2 and knead for 3 minutes
+   - Keep an eye on the mixture. If it looks dry, sprinkle in more water 1 tbsp at a time.
+     - Whether or not the dough needs more water will depend on your flour and the humidity in the air. It's not an exact science.
+   - The dough should form a ball, pull away from the sides of the bowl, and leave no excess flour in the bowl.
```
I think it'd separate things out nicely if you put a space between this bullet point and the image. Same with the other images in your procedures

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
-Repeat until all noodles have been cut
+1. Divide dough into 4 equal(ish) portions.
+2. Select one portion to work with and re-wrap the other 3. 
+   > **Note**: You can decide if you'd rather sheet and cut the whole batch at once, or sheet and cut one portion at a time. This guide does the latter.
```
I like the note! Nice formatting

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
+7. Lightly sprinkle flour over both sides of the dough to ensure it won't stick.
+8. Feed the dough through the sheeter.
+9. Fold it in half, and feed it through the sheeter again.
+10. Repeat steps 6 - 9 for Setting 3 and 4.
```
this is a lot of listed items at once. Could they be broken up in some way?

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
+
+#### Store Additional Pasta
+1. Toss pasta with extra flour, and allow it to dry for at least 24 hours or untill fully dry to the touch
+   - Exact times will vary depending ont he humidity in the room.
```
I like how you broke this up

gmsimon22 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -4,48 +4,66 @@ Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better
+#### Store Additional Pasta
+1. Toss pasta with extra flour, and allow it to dry for at least 24 hours or untill fully dry to the touch
+   - Exact times will vary depending ont he humidity in the room.
+2. Store in an airtight container in a dry, dark place for up to 1 month.
```
Great job!

WBlan97 replied:
I second that! I think both of these sections are very good.

alyssajohnsonunofficial replied:
Third! These sections are great and you've really got them fleshed out already!

gmsimon22 reviewed (commented):
Left some reviews for this procedure! Looks great!

WBlan97 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,76 @@
+# Making Homemade Pasta
+Use the revolutionary Stand'o'Mix 300 stand mixer to make homemade pasta! Better than storebought, and cooks in half the time.
```
I would consider adding a bit more staging information here. It is a lighthearted, fun procedure, and I think even something like a personal anecdote could work well! I'd also hyperlink the Stand'O'Mix 300 to their product website or maybe a rankings list like this: [Stand o mix ranking](https://www.foodandwine.com/lifestyle/kitchen/best-stand-mixersurl).

WBlan97 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,76 @@
+- 3 1/2 cups All-purpose Flour
+- 1/2 tsp Salt
+- 1/4 cup water 
+
```
Woah big recipe! I think the ingredients and tools are laid out very nicely.

alyssajohnsonunofficial replied:
Great layout here! I'd note the typo for the word 'yields', and I'd recommend capitalizing the first letter of every word in the ingredients list to be consistent! 'Cups', 'Tsp', Cup', etc!

WBlan97 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,76 @@
+4. Knead for 3 minutes on speed 2.
+   - Keep an eye on the mixture. If it looks dry, sprinkle in more water 1 tbsp at a time.
+     - Whether or not the dough needs more water will depend on your flour and the humidity in the air. 
+   - The dough should form a ball, pull away from the sides of the bowl, and leave no excess flour in the bowl.
```
Great alerting moves here!

alyssajohnsonunofficial replied:
Agreed, I really like your alerting moves!

WBlan97 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,76 @@
+2. Select one portion to work with and re-wrap the other 3. 
+   > **Note**: It is your choice whether to sheet and cut the whole batch at once, or sheet and cut one portion at a time. This guide does the latter.
+3. Flatten the dough until it's about a half-inch thick.
+4. Set the stand mixer speed to 2 and carefully feed the dough into the pasta sheeter. 
```
Maybe bold/italicize 2 here.

WBlan97 commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,76 @@
+10. Fold it in half, and feed it through the sheeter two more times.
+11. Repeat steps 7 - 10 for Settings 3 and 4.
+     - If the sheet of pasta gets too long to handle, cut in half or thirds.
+
```
Great job here! I think the language is clear and instructive, with good use of imperative verbs. Because there are 11 tasks, I'd maybe consider breaking them up somehow with an unordered list.

WBlan97 commented:
Great job! I left some comments, hope they're helpful.

dmbrasw2 committed:
Applied peer review edits

Applied edits suggested in peer review

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,74 @@
+# Making Homemade Pasta
+Homemade pasts is tastier than storebought, and cooks in half the time. And the stand'o'mix makes the process a breeze: kneading the dough, rolling it out into a nice, even sheet, and even cutting the noodles!
```
I really like this introduction! It's pretty concise and straightforward, but it's still got a tone that feels appealing and inviting (especially for someone new to this). Did want to note the typo in the second word of the intro ("pasts" instead of "pasta"), but you probably already caught that!

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,74 @@
+## Let's Make Pasta!
+
+### Prepare!
+1. Measure out the flour, salt, and water, and set out the eggs. 
```
Great use of imperative verbs!

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-pasta.md:
```diff
@@ -0,0 +1,74 @@
+     - Whether or not the dough needs more water will depend on your flour and the humidity in the air. 
+   - The dough should form a ball, pull away from the sides of the bowl, and leave no excess flour in the bowl.
+
+![Pasta dough pulling away from the bowl](<assets/images/screenshots/Pasta pics/Pasta dough.jpg>)
```
Great images, super appetizing and I like how you've got them dispersed throughout the procedure!

alyssajohnsonunofficial reviewed (commented):
This looks great, Deanna! Plenty of content already, and you've really nailed the right tone and style for your intended audience!

dmbrasw2 committed:
Spellcheck

Corrected spellings per peer review

dmbrasw2 committed:
User scenario copy + Figure #s

Copied the User Scenario.md and renamed to be specific for this procedure

Added figure labels/numbers + descriptions for images

dmbrasw2 committed:
User scenario fix

Copied user scenario wrong the first time. Deleted the miscopied file

dmbrasw2 committed:
Added user scenario

Included user scenario for make pasta

lingeringcode committed:
Feedback for Deanna

dmbrasw2 committed:
Applied changes per feedback

Numbered headings, specified 'tip' and 'note' in a few places, and reworded some steps
